bindings/web5_uniffi_wrapper/src/errors.rs

```diff
@@ -148,6 +122,8 @@ impl From<Web5Error> for InnerWeb5Error {
             return InnerWeb5Error::Crypto(msg);
         } else if variant == variant_name(&InnerWeb5Error::Encoding(String::default())) {
             return InnerWeb5Error::Encoding(msg);
+        } else if variant == variant_name(&InnerWeb5Error::Mutex(String::default())) {
+            return InnerWeb5Error::Mutex(msg);
```

Comment:
This if tree is a little tough to read. What do you think of using a match statement for this method?

```rust
    fn from(error: Web5Error) -> Self {
        let variant = error.variant();
        let msg = error.msg();

        match variant.as_str() {
            "Json" => InnerWeb5Error::Json(msg),
            "Parameter" => InnerWeb5Error::Parameter(msg),
            "DataMember" => InnerWeb5Error::DataMember(msg),
            "NotFound" => InnerWeb5Error::NotFound(msg),
            "Crypto" => InnerWeb5Error::Crypto(msg),
            "Encoding" => InnerWeb5Error::Encoding(msg),
            "Mutex" => InnerWeb5Error::Mutex(msg),
            _ => InnerWeb5Error::Unknown(format!("unknown variant {} with msg {}", variant, msg)),
        }
    }
```

Reply:
nice! done